data.sql

```diff
@@ -17,4 +17,60 @@ VALUES ('Charmander', '2020-02-08', -11, false, 0, ''),
        ('Boarmon', '2005-06-07', 20.4, true, 7, ''),
        ('Blossom', '1998-10-13', 17, true, 3, ''),
        ('Ditto', '2022-05-14', 22, true, 4, '');
-       
\ No newline at end of file
+
+
+-- Project 3
+
+INSERT INTO owners (full_nam, age)
+VALUES 
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+
+INSERT INTO species (name)
+VALUES ('Pokemon'), ('Digimon');
+
+
+ALTER TABLE animals
+SET species_id = 2, owner_id = 1
+WHERE name = 'Agumon';
+
```

Comment:
- [ ] As you may notice in your attached picture, how all the animal's name ending with `mon` doesn't have `Digimon` as a `species_id`, and all other animals have Pokemon instead, Kindly make sure you fix that.

Hint: To achieve that you may consider saying:
```sql
UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon') -- Or you can hardcode the species_id here which I wouldn't recommend --
   WHERE name like '%mon';

   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

Reply:
- [ ] This being said, you may consider assigning each owner to their animals like so:
```sql
UPDATE animals
 SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
-- With more animals  you may say for example--

UPDATE animals
 SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
  WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
```